### ISSUE TITLE
Quick links overlap gene list in advanced mode

### ISSUE THREAD

**Issue by CuzickA** [pombase/canto#1981]
![image](https://user-images.githubusercontent.com/11753634/61297646-8e129a80-a7d4-11e9-884c-bfa67f2b59cb.png)
![image](https://user-images.githubusercontent.com/11753634/61297683-9d91e380-a7d4-11e9-8abf-befee9c95504.png)

@jseager7 please could you look into this as discussed. Thanks

**Comment by jseager7** [comment]
This bug is more prone to happening when the screen is narrow. I think the layout is prevented from wrapping due to the use of tables in the gene list. There's a few ways to solve this:

### Option 1: Stack genes vertically

Rather than the genes expanding to the right, we could expand downwards instead. This solves the width issues at the cost of more height, potentially pushing the genotype links and annotation tables off-screen in the worst case.

To save some height, we could use a two-column design with pathogens in one column and hosts in the other. That's the design shown below:

![narrow_gene_table](https://user-images.githubusercontent.com/37659591/61625815-88550300-ac73-11e9-9ae4-c966b63e190a.PNG)

This design is more consistent with how single organism mode displays:

![image](https://user-images.githubusercontent.com/37659591/61625965-eaae0380-ac73-11e9-8dcb-fe498dae3269.png)

### Option 2: Move the 'Publication details' section

In case Option 1 doesn't solve the width issue on smaller screens, another option is to reclaim horizontal space by moving the publication details somewhere else on the page. My suggestion is to have the publication title and/or authors as a subheading of the page (we don't necessarily need the PMID, since that's already in the page heading).

![pmid_heading](https://user-images.githubusercontent.com/37659591/61626581-8724d580-ac75-11e9-8197-8c831c5ad890.PNG)

(Note the mockup I've shown doesn't have a link to the PubMed page, but that could easily be added.)

For maximum width savings, we could combine Option 2 with the alternative table design in Option 1.

This option is more disruptive than Option 1, since it affects all Canto modes. I'm keen to hear from @ValWood and/or @mah11 about whether they think this is any better than what we're currently doing.

**Comment by ValWood** [comment]
I don't have a strong preference, but I think I prefer vertical.

**Comment by jseager7** [comment]
@kimrutherford I tried to fix this by allowing the genes to wrap &ndash; which I did by collapsing all the gene links into one cell &ndash; but this didn't solve the problem. As I suspected, the real root of the problem is the fact we're using tables, because the containing `<div>` resizes correctly...

![gene_table_div_sizing](https://user-images.githubusercontent.com/37659591/62127347-bd8fcf80-b2c9-11e9-973f-4a253e41753e.png)

But the `<table>` doesn't:

![gene_table_table_sizing](https://user-images.githubusercontent.com/37659591/62127354-c1bbed00-b2c9-11e9-8ab5-dfbb563472cd.png)

I'm guessing something must be forcing the table cells to use a fixed width. That might be time-consuming to fix (unless there's some cheap `flex` hack), so I'll investigate this later.
